x-pack/plugins/security_solution/server/lib/siem_migrations/rules/api/index.ts

```diff
@@ -30,6 +32,8 @@ export const registerSiemRuleMigrationsRoutes = (
   registerSiemRuleMigrationsRetryRoute(router, logger);
   registerSiemRuleMigrationsStatsRoute(router, logger);
   registerSiemRuleMigrationsStopRoute(router, logger);
+  registerInstallMigrationRulesRoute(router, logger);
+  registerInstallAllMigrationRulesRoute(router, logger);
```

Comment:
Can we stay consistent with the rest of the routes names?
```suggestion
  registerSiemRuleMigrationsInstallRoute(router, logger);
  registerSiemRuleMigrationsInstallTranslatedRoute(router, logger); 
```
(assuming we rename `install_all` to `install_translated`)

Reply:
done